### DISCUSSION TITLE
In the setup() function numeric_imputatin = "drop" does not work

### DISCUSSION THREAD

**Discussion by kgmat** [pycaret/pycaret#3498] [General]
Hi, i am trying to setup an experiment like this:

setup(data=df, target="tax", session_id=123, **numeric_imputation="drop"**)

but, it does not drop the rows with empty cells, and keeps on raising the ValueError 

**ValueError: 8 missing values found in the target column: tax. To proceed, remove the respective rows from the data**

is there anything i am doing wrong, any help/guidance is highly appreciated.

thanks

**Comment by ngupta23** [comment]
Your target column "tax" has missing values. The target column can not have missing values. You can remove those rows that have missing values in the "tax" column and proceed.
